io/src/main/java/org/commonjava/maven/ext/manip/rest/DefaultVersionTranslator.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2012 Red Hat, Inc. (jcasey@redhat.com)
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
  * Unless required by applicable law or agreed to in writing, software
```

Comment:
Can we maintain the same license format as other files as otherwise the licensing check will fail